_docs/_latest/setup/install.md

```diff
@@ -22,14 +22,33 @@ organization Cloud IAM policy.
   Forseti 1.0 installed in it.
 * Enable billing on the project.
 
-## Setting up Forseti Security
+## **Setting up Forseti Security**
 
 The Forseti Terraform module is the only supported method of installing Forseti Security. The default infrastructure for 
 Forseti is Google Compute Engine. This module also supports installing Forseti on Google Kubernetes Engine (GKE), 
 and at some point in the future will become the default. For more information on installing Forseti on-GKE, please see 
 the [detailed guide on setting up Forseti on-GKE]({% link _docs/latest/setup/forseti-on-gke.md %}).
 
-## Google Cloud Shell Walkthrough
+## **Requirements**
+This section describes in detail the requirements necessary to deploy Forseti. The setup helper script automates the 
+service account creation and enabling the APIs for you. Read through this section if you are not using the setup script 
+or want to understand these details.
+
+1. Install Terraform.
+2. A GCP project to deploy Forseti into. The 
+[Google Project Factory Terraform](https://github.com/terraform-google-modules/terraform-google-project-factory) module 
+can be used to provision the project with the required APIs enabled, along with a Shared VPC connection.
+3. The Service Account used to execute this module has the right permissions.
+4. Enable the required GCP APIs to allow the Terraform module to deploy Forseti.
+
+## **Software Dependencies**
+
+### Terraform and Plugins
```

Comment:
(nit) I know this PR is just about reorganizing the docs, but I don't think we need to list the provider versions on the website. The user don't need to do anything to get those provider versions and as we update provider versions, we will need to update the website.

Reply:
I too think `Software Dependencies` section is not required. I went ahead and removed it in this PR itself.